include/ConfigManager.h

```diff
@@ -219,6 +218,10 @@ class LMMS_EXPORT ConfigManager : public QObject
 		return m_version;
 	}
 
+	// Used when the configversion attribute is not present in a configuration file.
+	// Returns the appropriate config file version based on the LMMS version.
+	const unsigned int legacyConfigVersion();
```

Comment:
Did you mean `unsigned int legacyConfigVersion() const;`? (Same in `DataFile.h`.)